.github/workflows/pull_request.yml

```diff
@@ -120,27 +148,13 @@ jobs:
 
       - name: Set up infra for integration test
         run: |
+          OPENEBS_NAMESPACE="openebs"
           sudo apt-get update && sudo apt-get install -y xfsprogs quota linux-modules-extra-`uname -r`
           go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@v2.14.0
-          sed -e '/openebs-provisioner-hostpath/{N;s/IfNotPresent/Never/}' deploy/kubectl/openebs-operator-lite.yaml | kubectl apply -f -
+          helm install localpv-provisioner ./deploy/helm/charts -n $OPENEBS_NAMESPACE --create-namespace --set localpv.image.pullPolicy=Never --set analytics.enabled=false
+          kubectl get pods -A
 
       - name: Integration test
         run: |
           make integration-test
 
```

Comment:
Please add code coverage for the integration tests as well.

ref:- https://github.com/openebs/zfs-localpv/blob/b2f57b9b078379681277565d81c4d46a9acd2b10/ci/ci-test.sh#L78

Please make sure the file path is correct

Reply:
I added that in the makefile itself. Is that not correct ?

Reply:
The makefile is for contributors, and CI workflows.
However, to coverage reports are expected to come from the CI exclusively. It shouldn't live inside the makefile.

Reply:
In that case, where should it go ? We don't have sh scripts calling tests in this repo. They are being executed using make.

Reply:
Codecov step is missing

Reply:
Also, seems like its not working properly. I see errros on the codecov portal. So that also needs to be fixed.

Reply:
Added the codecov upload step. On the code cov portal errors, they would need further investigation. This is not working for other repos too. I will raise an issue to have it debugged and fixed.